src/main/java/seedu/exercise/ui/ResolveWindow.java

```diff
@@ -34,7 +35,7 @@
 
     public ResolveWindow(Stage root) {
         super(FXML, root);
-        blockEvents(root);
```

Comment:
Do remember to add this line back to block access to the parent window when the resolve window pops up.

Reply:
done